src/components/Card.tsx

```diff
@@ -1,9 +1,13 @@
+/* eslint-disable react/jsx-no-comment-textnodes */
 import clsx from 'clsx';
 import type { Props } from '$util/types';
 
+//* Card takes in contents and returns the full Card */
 export default function Card({ children, className, ...props }: Props): JSX.Element {
 	return (
+		//* Main div section containing props */
```

Comment:
This comment is not valid, it will be rendered! Could wrap it in curly braces.

Reply:
it is, wrapping it in curly braces throws an error